SPECS/selinux-policy/0004-Set-default-login-to-unconfined_u.patch

```diff
@@ -1,20 +1,23 @@
-From 2cb00d4d4bf0919ad55b9c3804f0546b5e6db13c Mon Sep 17 00:00:00 2001
+From f8a5540f52d0e190c6321cd40e6c1edb058637bc Mon Sep 17 00:00:00 2001
 From: Chris PeBenito <Christopher.PeBenito@microsoft.com>
 Date: Wed, 1 Sep 2021 20:30:29 +0000
-Subject: [PATCH 4/8] Set default login to unconfined_u.
+Subject: [PATCH 04/15] Set default login to unconfined_u.
```

Comment:
Nit: Patch number is wrong here + will be wrong in unmodified patches. Feel free to ignore this though as it impacts nothing.